### DISCUSSION TITLE
[Doc] Auto-split Hash Range:：C1 to C3 ？？

### DISCUSSION THREAD

**Discussion by startjava** [apache/pulsar#19040] [Q&A]
### Search before asking

- [X] I searched in the [issues](https://github.com/apache/pulsar/issues) and found nothing similar.

### What issue do you find in Pulsar docs?

https://pulsar.apache.org/docs/next/concepts-messaging/#subscription-modes

old：
C1？
Given a message key Order-3459134, its hash would be murmur32("Order-3459134") = 3112179635, and its index in the range would be 3112179635 mod 65536 = 6067. That index is contained within region [0, 16384) thus consumer C1 will be mapped to this message key.

new：
C3？？

@momo-jun 

### What is your suggestion?

?

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

**Comment by tisonkun** [comment]
Moved to https://github.com/apache/pulsar/discussions/categories/q-a since it's more on a question.

**Comment by momo-jun** [comment]
Hi @asafm, Happy holidays! Can you pls take a look at this question when you are back from the holidays? Thank you.

**Reply by asafm** [reply]
Thanks, @momo-jun, for the reference. Thanks, @startjava, for spotting the error. I've also fixed it in this PR: https://github.com/apache/pulsar/pull/19063/files
